ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -28,7 +28,7 @@
          data-sly-call="${allowedTemplate.allowedcomponents @ title=allowed.title, components=allowed.components}"></sly>
     <input type="hidden" name=":formstart" value="${resource.path}"/>
     <input type="hidden" name="_charset_" value="utf-8"/>
-    <input data-sly-test="${container.thankyouPage}" type="hidden" name=":redirect" value="${container.thankyouPage @ extension='html'}"/>
+    <input data-sly-test="${container.redirect}" type="hidden" name=":redirect" value="${container.redirect @ extension='html'}"/>
```

Comment:
I don't see a getRedirect method in the interface.

Reply:
changed to redirectUrl. Thanks for pointing it out!